appinfo/info.xml

```diff
@@ -15,6 +15,9 @@
 	<website>https://github.com/matiasdelellis/facerecognition</website>
 	<bugs>https://github.com/matiasdelellis/facerecognition/issues</bugs>
 	<repository type="git">https://github.com/matiasdelellis/facerecognition.git</repository>
+	<types>
```

Comment:
so, this was main issue?:)

Reply:
Yes.. Please, look my comments that I just summit..